mediq/settings.py

```diff
@@ -48,9 +48,11 @@
 
     # third-party apps
     'rest_framework',
+    'rest_framework_simplejwt',
 
     # mediq apps
     'accounts',
+    'organizations',
```

Comment:
hi @vbello-tech 
let's use british english across the app. "s" instead of "z"
oraganisation, instead of organization

Reply:
ignore this. i can see you've used that across the entire application. we'll stick with american english